src/protocols/multisig/mod.rs

```diff
@@ -0,0 +1,157 @@
+/*
+    Multisig Schnorr
+
+    Copyright 2018 by Kzen Networks
+
+    This file is part of Multisig Schnorr library
+    (https://github.com/KZen-networks/multisig-schnorr)
+
+    Multisig Schnorr is free software: you can redistribute
+    it and/or modify it under the terms of the GNU General Public
+    License as published by the Free Software Foundation, either
+    version 3 of the License, or (at your option) any later version.
+
+    @license GPL-3.0+ <https://github.com/KZen-networks/multisig-schnorr/blob/master/LICENSE>
+*/
+
+//! Schnorr {n,n}-Signatures based on Accountable-Subgroup Multisignatures
+//!
+//See (https://pdfs.semanticscholar.org/6bf4/f9450e7a8e31c106a8670b961de4735589cf.pdf)
+use cryptography_utils::elliptic::curves::traits::*;
+use cryptography_utils::{BigInt, FE, GE};
+
+use cryptography_utils::cryptographic_primitives::hashing::hash_sha256::HSha256;
+use cryptography_utils::cryptographic_primitives::hashing::traits::*;
+use protocols::multisig;
+#[derive(Debug, Clone)]
+pub struct Keys {
```

Comment:
Please document the role of `I` and `X` in the protocol below (if possible, add a link to their definition in the paper).